mandatory/3-function-output.js

```diff
@@ -2,13 +2,18 @@
 function getRandomNumber() {
   return Math.random() * 10;
 }
+//This function is returning a random number between 0  and 10 
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
+//The concat() method joins two or more strings together into a new string without modifying the original strings
 
 function concatenate(firstWord, secondWord, thirdWord) {
+  return firstWord + " "+ secondWord + " "+thirdWord;
```

Comment:
Question: Have we answered the question here? I think the goal is to use the concat string method here. 

i.e. 

```
const wordOne = "Hello"
const wordTwo = "World"

const combinedWord = wordOne.concat(wordTwo)
```

How would you concat three words like above?